sunpy/map/map_factory.py

```diff
@@ -4,12 +4,13 @@
 import glob
 from collections import OrderedDict
 import warnings
-
 import numpy as np
 import astropy.io.fits
+from astropy.wcs import WCS
 
 import sunpy
-from sunpy.map.mapbase import GenericMap
+from sunpy.data.sample import AIA_171_IMAGE
```

Comment:
If this is only used in the example, it should be imported there.

Reply:
I did this but only 2 out of 6 checks are successful

Reply:
Ah I see why. You need to add ```# doctest: +REMOTE_DATA``` to any line that uses the sample data.

Reply:
Also it is already imported in that example. as ```sunpy.data.sample.AIA_171_IMAGE```